examples/examples.ipynb

```diff
@@ -2,41 +2,20 @@
  "cells": [
```

Comment:
(action not required) I guess this notebook file is no longer necessary as we could provide a comprehensive examples in Google Colab. I will remove this file later.